src/Illuminate/Foundation/Console/stubs/config.stub

```diff
@@ -0,0 +1,14 @@
+<?php
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | {{ class }} Config File
+    |--------------------------------------------------------------------------
+    |
+    | This file is for storing the credentials of the {{ class }} application
```

Comment:
Typically one would put these in the services.php file, so this doesn't make for a good example. Moreover, I think it's rare app developers would want to make another config file, since the main use case for config is exactly the one you identified - and that is already serviced by the services.php config file (pun intended :P).

Reply:
Yes, it's true but of course the moment that developer want to create config file will be occured. Since laravel ecosystem includes lots of make command, the config file should be included. That was the main purpose.